code/game/objects/items/devices/taperecorder.dm

```diff
@@ -91,8 +91,16 @@
 
 /obj/item/taperecorder/hear_talk(mob/M, list/message_pieces, verb)
 	var/msg = multilingual_to_message(message_pieces, requires_machine_understands = TRUE, with_capitalization = TRUE)
+	var/voice = "Unknown"
+	if (M.type == /mob/living/carbon/human)
+	{
+		var/mob/living/carbon/human/H = M
+		voice = H.voice
+	}
+	else
+		voice = M.name

```

Comment:
```suggestion
	var/voice = M.name ? M.name : "Unknown"
	if (ishuman(M))
		var/mob/living/carbon/human/H = M
		voice = H.voice
```
DM is non-context-free, you don't need the curly braces.

Reply:
@Sharkmare bumping this, you gonna get back to this?

Reply:
for some reason i only got the bump as a notification odd